defi/src/cli/fillOld.ts

```diff
@@ -107,12 +111,12 @@ const main = async () => {
   }
   let dailyItems: any = []
 
-  if (!IS_DRY_RUN)
+  /* if (!IS_DRY_RUN)
```

Comment:
since youre disabling dry runs add some code like

if(IS_DRY_RUN){
  throw new Error("Dry runs are no longer supported")
}

that way if someone tries to do this in the future they will get that error

Reply:
IS_DRY_RUN is still supported, what I am doing here is skipping data pull from ddb since we pass a flag to overwrite data.

That brings me to a question I forgot to ask, if we write data to the same PK & SK, it overwrites in ddb?

Reply:
>  if we write data to the same PK & SK, it overwrites in ddb?

yes